Fix #2352: Update Ktlint version to "0.37.1"

srikanthkb opened the pull request:
<!--
  - Thanks for submitting code to Oppia! Please fill out the following as part of
  - your pull request so we can review your code more easily.
  -->

## Explanation
<!--
  - Explain what your PR does. If this PR fixes an existing bug, please include
  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
  - when this PR is merged.
  -->
Fixes #2352

Updated the Ktlint download version in github workflows from "0.37.0" to 0.37.1"

## Checklist
<!-- Please tick the relevant boxes by putting an "x" in them. -->
- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

srikanthkb committed:
Fix #2352: Update Ktlint version to "0.40.0"

Fixes #2352

Updated the Ktlint download version in workflows
from "0.37.0" to 0.40.0"

FareesHussain commented on .github/workflows/main.yml:
```diff
@@ -27,7 +27,7 @@ jobs:
     - name: Download Ktlint
       run: |
-        KTLINT="0.37.0"
+        KTLINT="0.40.0"
```
As there are few errors with this version can you try 0.37.1

as it has a windows fix in that version

FareesHussain replied:
reference - https://github.com/pinterest/ktlint/issues/761#issue-630042183

srikanthkb replied:
@FareesHussain Thank you for the suggestion, I will confirm with 0.37.1 and update.

anandwana001 replied:
resolving on behalf of @FareesHussain

FareesHussain reviewed (changes requested):
@Srikanth-Kb PTAL

srikanthkb committed:
Fix #2352: Update Ktlint version to "0.37.1"

Fixes #2352

Updated the Ktlint download version in workflows
from "0.37.0" to 0.37.1". This version runs successfully
on Windows 10 OS.

anandwana001 reviewed (approved):
LGTM, Thanks @Srikanth-Kb 
Also, please update the issue and pr title for 0.37.1

FareesHussain reviewed (approved):
LGTM, thanks

srikanthkb commented:
> LGTM, Thanks @Srikanth-Kb
> Also, please update the issue and pr title for 0.37.1

srikanthkb commented:
> LGTM, Thanks @Srikanth-Kb
> Also, please update the issue and pr title for 0.37.1

srikanthkb commented:
Done.

rt4914 reviewed (approved):
LGTM, thanks.
